### PR TITLE
Updated new milestones and deliverables

### PR DESCRIPTION
**Pull request by jkosem**

### CODE REVIEW

File: targeted-grants/IPFS-mobile-design-research.md [hunk 1]
```diff
@@ -31,31 +30,36 @@ The following is the two phase IPFS Mobile Design Guidelines plan. All of the re
...
-
-Deliverables
-
-- Survey review of existing interfaces, components and patterns
-- Documentation of research in GitHub or other appropriate online repository
-- Design strategy
-- Research findings report to become part of the final report with the completion of the design phase
-- Writing and publication of blog post summarising the project
-- Presentation of the project at IPFS Weekly
+26 Feb 2020: Proposed date to start
```
Comment (momack2) [inline targeted-grants/IPFS-mobile-design-research.md]: Between these high-level phases and the more detailed programme of work, can we tie each milestone to the deliverables mentioned below? It's not clear to me right now what deliverable / output will unlock the next milestone (survey review, interview report, ??)

Comment (jkosem) [reply targeted-grants/IPFS-mobile-design-research.md]: Working on it right now. Will preview and vett it's okay before another PR with @autonome

Comment (momack2) [reply targeted-grants/IPFS-mobile-design-research.md]: Thanks!

**Comment by jkosem** [comment]
> Thank you! This is gives much more clarity.

See the last edit, it should be even clearer now. let me know. Thanks.

**Pull request merged**